src/main/java/com/redhat/developers/msa/aloha/AlohaVerticle.java

```diff
@@ -114,15 +113,19 @@ private void alohaChaining(Handler<List<String>> resultHandler) {
      * @return The feign pointing to the service URL and with Hystrix fallback.
      */
     private BonjourService getNextService() {
+    	String zipkinUrl = System.getenv("ZIPKIN_SERVER_URL");
```

Comment:
I think we should declare this ZIPKIN_SERVER_URL inside the Dockerfile. I don't think this service can be used outside a container since it needs a ZipKin running